initfiles/bin/init_roxie

```diff
@@ -40,9 +40,6 @@ fi
 export LIBC_FATAL_STDERR_=1
 export restarts=0
 
-ulimit -c unlimited
-ulimit -n $NUM_ROXIE_HANDLES
```

Comment:
NUM_ROXIE_HANDLES now gone..

If we really don't care.. then we should also remove the configuration option from the configxml xslt files.